src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataServiceTest.java

```diff
@@ -152,7 +138,7 @@ public void shouldCreateCase() throws ApplicationExceptions.EntityCreationExcept
 
         when(caseDataRepository.getNextSeriesId()).thenReturn(caseID);
         when(infoClient.getCaseDeadline(caseType.getDisplayCode(), deadlineDate, 0)).thenReturn(caseDeadline);
-        when(infoClient.getCaseType(caseType.getDisplayCode())).thenReturn(caseType);
```

Comment:
Where we were calling infoClient directly we have repointed the code to call the new CaseDataTypeService.